src/web/app/pages-instructor/instructor-course-student-edit-page/instructor-course-student-edit-page.component.ts

```diff
@@ -128,6 +136,23 @@ export class InstructorCourseStudentEditPageComponent implements OnInit, OnDestr
             .subscribe(() => this.isEmailFieldChanged = true);
   }
 
+  /**
+   * Displays message to user stating that the field is empty.
+   */
+  displayEmptyFieldMessage(fieldName: string, maxLength: number): string {
+    return `The field '${fieldName}' is empty.` +
```

Comment:
I think `'${fieldName} should not be empty.` or `Please enter a ${fieldName}.` would be sufficient.
My reasoning is that when the field is empty, the user doesn't need to care about the `maxLength` (yet) until he actually exceeds the max length, which will trigger `displayExceedMaxLengthMessage`. A shorter message would also reduce cognitive load. What do you think?

Reply:
@monmanuela That's a valid point. Most of the messages included here were referenced from `FieldValidator.java` (back-end validation). 

A shorter and more succinct message would be more suitable for front-end validation. We might want to keep this in mind and improve the back-end validation message if appropriate.